opennav_coverage_navigator/src/coverage_navigator.cpp

```diff
@@ -23,54 +23,46 @@ namespace opennav_coverage_navigator
 
 bool
 CoverageNavigator::configure(
-  rclcpp_lifecycle::LifecycleNode::WeakPtr parent_node,
+  nav2::LifecycleNode::WeakPtr parent_node,
   std::shared_ptr<nav2_util::OdomSmoother> odom_smoother)
 {
   start_time_ = rclcpp::Time(0);
   auto node = parent_node.lock();
 
-  if (!node->has_parameter("path_blackboard_id")) {
-    node->declare_parameter("path_blackboard_id", std::string("path"));
-  }
-  path_blackboard_id_ = node->get_parameter("path_blackboard_id").as_string();
+  path_blackboard_id_ = node->declare_or_get_parameter(
+    getName() + ".path_blackboard_id", std::string("path"));
```

Comment:
I don't think the getName() is necessary and changes the name it looks for parameters under

Reply:
That is on purpose, see the PR description.
Though I would understand if this should be in a separate PR or need some discussion

Reply:
reverted